spec/lib/pivotal-tracker/following_spec.rb

```diff
@@ -1,16 +1,9 @@
 require 'spec_helper'
 
 describe PivotalTracker::Following do
-  context '#new' do
-    it 'initializes with attributes' do
-      attributes = {id: 1, person_id: 1, story_id: 1, epic_id: 1, kind: 'kind'}
-      following = PivotalTracker::Following.new(attributes)
-
-      expect(following).to be_a PivotalTracker::Following
-
-      attributes.each do |attribute, value|
-        expect(following.send(attribute)).to eq value
-      end
-    end
+  it_behaves_like "an API backed model" do
+    let(:attributes) { FactoryGirl.attributes_for(:following) }
+    let(:subject) { PivotalTracker::Following.new(attributes) }
```

Comment:
@jasonnoble I am going to do this now in all the specs and wrap it into one commit